chroot-etc/cmt-xorg.conf

```diff
@@ -0,0 +1,3 @@
+Section "Server Flags"
```

Comment:
cmt only works with `xorg`, so how about modifying chroot-etc/xserverrc-xorg to add -ignoreABI to the flags?